terraform/basic_components/main.tf

```diff
@@ -73,6 +73,10 @@ data "template_file" "otconfig" {
     testing_id = var.testing_id
     grpc_port = module.common.grpc_port
     udp_port = module.common.udp_port
+    cortex_instance_endpoint = module.common.cortex_instance_endpoint
```

Comment:
it will be a fixed value? if we put it in the output of common module, we can't use var to change its value while we use var to change the value in github workflow. can we configure it in variables?